api/reviews_api.py

```diff
@@ -72,9 +70,16 @@ def do_post(self, **kwargs) -> dict[str, str]:
     self.require_permissions(user, fe, gate, new_state)
 
     # Note: We no longer write Approval entities.
-    approval_defs.set_vote(feature_id, None, new_state,
+    new_gate_state = approval_defs.set_vote(feature_id, None, new_state,
         user.email(), gate_id)
 
+    # Notify that trial extension has been approved.
+    if (gate.gate_type == GATE_API_EXTEND_ORIGIN_TRIAL and
+        new_gate_state == Vote.APPROVED):
```

Comment:
You should treat either `Vote.APPROVED` or `Vote.NA` as approval.

Also, it is possible that there could be more votes after the one needed for approval.  E.g., two reviewers might both have a browser tab open on the same gate and both approve it without reloading.  So, checking that `gate.state not in (Vote.APPROVED, Vote.NA)` would avoid sending a double email in that situation.